dotcom-rendering/src/palette.ts

```diff
@@ -6212,6 +6266,18 @@ const paletteColours = {
 		light: keyEventTitleLight,
 		dark: keyEventTitleDark,
 	},
+	'--kicker-background-live': {
+		light: liveKickerBackgroundLight,
+		dark: liveKickerBackgroundDark,
+	},
+	'--kicker-pulsing-dot-live': {
+		light: liveKickerPulsingDot,
+		dark: liveKickerPulsingDot,
+	},
+	'--kicker-text-live': {
+		light: liveKickerTextLight,
+		dark: liveKickerTextDark,
+	},
```

Comment:
Are these for cards exclusively? Is there a reason why we cannot simply override the `'--card-kicker-text'` in the `ContainerOverrides`?

Reply:
Just so I understand correctly:

1. Can a “live kicker” be used for anything other than an `ArticleDesign.LiveBlog`?
2. Can a “standard kicker” be used for an `ArticleDesign.LiveBlog`?

If no to both of the above, I think only `--card-kicker-text`. Similarly, for `--kicker-background-live` could become `--card-kicker-background` and return `transparent` in anything but an `ArticleDesign.LiveBlog`.

As mentioned, this can be addressed as a follow-up if more convenient if you need to move quickly on this :)